cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -103,7 +103,20 @@ set(configure_command
     ${configure_command})
 
 # Pass C flags through
-set(configure_opts ${configure_opts} ${CMAKE_C_FLAGS})
+if(APPLE)
+  # Apple cannot process `-fPIC` flag argument (PR-1121)
+  string(REGEX MATCHALL "[^ ]+" PARSED_C_FLAGS "${CMAKE_C_FLAGS}")
+  set(C_FLAGS_NO_PIC)
+  foreach(C_FLAG IN LISTS PARSED_C_FLAGS)
+    if(NOT C_FLAG MATCHES "^--?f?(PIC|PIE|pic|pie)$"
```

Comment:
I think I got a better idea. Instead of working with raw flag's string you can add `SKIP_PIC` to `hunter_dump_cmake_flags` to skip this code:
* https://github.com/ruslo/hunter/blob/00e5402b46c2f9090ddc7deb7bed9c4ed56530c5/cmake/modules/hunter_dump_cmake_flags.cmake#L123-L139

And apply `SKIP_PIC` only for OSX in scheme:
```cmake
set(skip_pic "")
if(APPLE)
  set(skip_pic SKIP_PIC)
endif()

hunter_dump_cmake_flags(... ${skip_pic})
```

Reply:
It will not work when user explicitly will set `-fPIC` to `CMAKE_CXX_FLAGS` but in this case it can be fixed in toolchain by using `set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)` instead.

Reply:
OK great. I am not so worried about that use case because `CMAKE_POSITION_INDEPENDENT_CODE` is the ideal way to set `-fPIC`.